src/Features/Core/Portable/Completion/CompletionRules.cs

```diff
@@ -160,7 +160,7 @@ public CompletionRules WithSnippetsRule(SnippetsRule snippetsRule)
         private static readonly ImmutableArray<char> s_defaultCommitKeys = ImmutableArray.Create(
                 ' ', '{', '}', '[', ']', '(', ')', '.', ',', ':',
                 ';', '+', '-', '*', '/', '%', '&', '|', '^', '!',
-                '~', '=', '<', '>', '?', '@', '#', '\'', '\"', '\\');
+                '~', '=', '<', '>', '?', '#', '\'', '\"', '\\');
```

Comment:
Why this change?

Reply:
The easiest way to see the bug was this scenario (even before my change). Start with this (caret at `$$`):

```
string @this;
string str;
str = $$
```

Then press <kbd>Ctrl</kbd>+<kbd>Space</kbd> followed by <kbd>@</kbd>. Completion should allow the `@` to be typed as the start of an identifier, but right now it cannot.

Reply:
➡️ This change was reverted.